src/content/experimental/ui-shell/usage.md

```diff
@@ -60,13 +78,24 @@ These panels are at the same elevation as the primary content zone of the page,
 
 ![Header](images/on-page-panel.png)
 
+Use an on-page panel when:
+
+- presenting additional information that is relevant to the page context (i.e., contextual help)
+- content does not need to always be on the page
+
 <!--### Footer
 Product footers are persistent and attached to the bottom of the browser window. A footer should be reserved for actions or information that is pertinent to the users current workflow. Footers should have clear means of dismissal. <mark>This is an unusual way to treat a footer, especially with it being dismissable... Is this really how we want to define them? -CJC</mark> <mark>Which "zone" does the footer belong to? - CJC</mark>
 
 ![Footer](images/footer-1.png) -->
 
 ## Side Navigation
```

Comment:
I think maybe we should call this section "Secondary Navigation". Its was confusing to me to have the section be called "Side Navigation" and then the only thing under it be "Side-nav". The two slightly different uses we're tripping me up. Especially since "side-nav" is mentioned a third time as "Side-nav Panels". I think we're over using the term manifestations for different meanings.